notion_client/errors.py

```diff
@@ -45,34 +69,69 @@ def is_http_response_error(e: Exception) -> bool:
 
 
 class APIErrorCode(str, Enum):
+    """Enumerate API error codes."""
+
     Unauthorized = "unauthorized"
+    """The bearer token is not valid."""
```

Comment:
I've never seen an Enum documented that way. Does it actually end up in `APIErrorCode.Unauthorized.__doc__` by some kind of magic, or is it just comments?

Reply:
It does not get into the `__doc__` attribute. But this is for mkdocstrings. Also, Microsoft python-language-server (that provides the hover and autocompletion in vs code), has a triaged open issue on this. https://github.com/microsoft/python-language-server/issues/1725